api/application_submissions/index.js

```diff
@@ -31,24 +31,18 @@ router.post('/', async (req, res) => {
   }
 });
 
-router.put('/:id', async (req, res) => {
-  const { id } = req.params;
+router.put('/', async (req, res) => {
```

Comment:
Here, I need to take in multiple id's. I currently read them in via the query URL but I am not sure if that follows the proper conventions. What should I do here?

Reply:
you can put them in the request body as an array of application IDs